v2/types.go

```diff
@@ -1494,13 +1494,19 @@ func NewTickerFromRaw(symbol string, raw []interface{}) (t *Ticker, err error) {
 	}
 	// funding currency ticker
 	// ignore bid/ask period for now
-	if len(raw) == 13 {
+	// on funding currencies (ex. fUSD)
+	// SYMBOL, FRR, BID, BID_PERIOD, BID_SIZE, ASK, ASK_PERIOD, ASK_SIZE, DAILY_CHANGE, DAILY_CHANGE_RELATIVE,
+	// LAST_PRICE, VOLUME, HIGH, LOW, _PLACEHOLDER, _PLACEHOLDER, FRR_AMOUNT_AVAILABLE
+	if len(raw) == 16 {
```

Comment:
For the sake of not breaking anything, can we keep the `if len(raw) == 13` as well?

```
if len(raw) == 13 {
    ...
} else if len(raw) == 16 {
    ...
}
```

Reply:
Sure, I updated the codes.